sign/sign_integration_test.go

```diff
@@ -0,0 +1,69 @@
+/*
+Copyright 2022 The Kubernetes Authors.
```

Comment:
Let's move this file into `/test` to be able to separately run integration and unit tests.

Reply:
Since we're doing this, do we also want to move other integration tests? I see that we have some integration tests for git.

Reply:
Yeah this sounds like a reasonable follow-up.